src/ConnectedMode.UnitTests/Binding/BindingInfoProviderTests.cs

```diff
@@ -32,6 +32,19 @@ namespace SonarLint.VisualStudio.ConnectedMode.UnitTests.Binding
     [TestClass]
     public class BindingInfoProviderTests
     {
+        [TestMethod]
+        public void MefCtor_CheckIsExported()
+        {
+            MefTestHelpers.CheckTypeCanBeImported<BoundConnectionInfoProvider, IBoundConnectionInfoProvider>(
+                MefTestHelpers.CreateExport<ISolutionBindingRepository>());
```

Comment:
Out of curiosity: What's the line length that you try to follow?

I saw that S103 is not active on SL. Should activate it?

Reply:
We don't have a rule for that. We decide what is more readable case by case. And I don't think it should be rule-based.

Reply:
Are you fine with long lines too? For example, we have S103 configured to 200 on sonar-dotnet, to have some guidance. And we ~~violate~~accept it where it makes sense.